explorer/src/components/SolanabeachComponents/BlockHeight.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from "react";
+import { Col, CardBody } from "reactstrap";
+import NumberFormat from "react-number-format";
+
+export default class BlockHeight extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data: 0,
+    };
+  }
+
+  componentWillUnmount() {
+    // Disconnect socket when component is unmounted
+    if (this.socket) {
+      this.socket.disconnect();
+    }
+  }
+
+  componentDidMount() {
+    const { socket, event } = this.props;
+
+    socket.on(event, (data) => this.setState({ data }));
+  }
+
+  render() {
+    return (
+      <Col md="6" lg="4">
+        <div className="card-box border-0">
+          <CardBody>
+            <div className="align-box-row align-items-start">
+              <div className="font-weight-bold">
+                <small className="ghostwhite d-block mb-1 text-uppercase">
+                  Block Height
```

Comment:
Unfortunately, Solana Beach is wrong here. This isn't the actual block height, it's the block slot number. Changing to "Block" or "Last Finalized Block" is better